packages/cli/lib/lib/webpack/webpack-client-config.js

```diff
@@ -146,7 +146,12 @@ async function clientConfig(env) {
 			}),
 			new PushManifestPlugin(env),
 			...(await renderHTMLPlugin(env)),
-			...getBabelEsmPlugin(env),
+			env.esm &&
+				new OptimizePlugin({
+					modernize: false,
+					minify: isProd,
+					exclude: [/^sw.*\.js/],
+				}),
```

Comment:
I was thinking, do we want to keep having users opt in/out with the `--esm` flag, or should this instead be something they can disable with their `preact.config.js` files?

The only real argument against outputting ESM too is, AFAIK, a desire to have fewer files/extreme disk space constraints. I don't think that's a common enough of a problem to continue to have this as a CLI flag.

Reply:
Hmm, so it looks like `optimize-plugin` is quite slow and doesn't play nice in dev. We'd have to remove the flag for `preact watch`, but prod is up in the air.

Reply:
Ended up removing, it's honestly simpler to have users cut out `optimize-plugin` in their config and adjust Terser's file matching. We can also push towards better practices.